config/locales/errors.en.yml

```diff
@@ -9,3 +9,7 @@ en:
       type: https://github.com/alphagov/account-api/blob/main/docs/api.md#unknown-attribute-names
       title: Unknown attribute names
       detail: Attribute names %{attributes} are unknown, have they been added to config/user_attributes.yml?
+    saved_page_not_found:
+      type: https://github.com/alphagov/account-api/blob/main/docs/api.md#saved_page_not_found
+      title: Saved page not found
+      detail: A saved page with that path cannot be found associated with that user, is the path correct?
```

Comment:
This should be the `cannot_save_page` error

Reply:
Also, for the other errors, the class name and i18n keys match the title, so it's dead simple to see what is related to what: `LevelOfAuthenticationTooLow => level_of_authentication_too_low => "Level of authentication too low"`

Reply:
Ok, now looking at the full diff, the problem is that these `saved_page_not_found` locale entries are introduced in the "create a saved page" commit, and the locale entries for the `CannotSavePage` class are not added anywhere.

Reply:
rebase-tastic... Sorry this thing has been rebased about 💯  x 💯 times now, it's easy for things to get lost along the way.
I'll get that back.